### DISCUSSION TITLE
tailwindcss/nesting not working

### DISCUSSION THREAD

**Discussion by xApep** [tailwindlabs/tailwindcss#9249] [Help]
tailwindcss: 3.1.8
postcss-nesting: 10.1.10

Installing postcss nesting as described here: https://tailwindcss.com/docs/using-with-preprocessors

In my style.css (imported in "tailwind.css") I have
```
svg {
  path {
    @apply fill-current;
  }
}
```

If I run a `npm run build...` (package.json job) it works fine.

but if after that I run it with 
`npx tailwindcss -i ./public/css/src/tailwind.css -o ./public/css/tailwind.css --watch`

it outputs as defined
![image](https://user-images.githubusercontent.com/1315340/188274210-25a85257-2a86-4513-9625-e43c03d57542.png)

Is there I way to get it to work?

**Comment by adamwathan** [comment]
Make sure you add the `--postcss` flag on the command line or your PostCSS config won’t be respected by Tailwind CLI.

**Reply by xApep** [reply]
That worked! Should be wise to doc it? :)

Thank you very much!
